svc-api/handle/telemetry.go

```diff
@@ -246,7 +247,6 @@ func (a *TelemetryRPCs) GetMetricReportDefinition(ctx iris.Context) {
 func (a *TelemetryRPCs) GetMetricReport(ctx iris.Context) {
 	req := telemetryproto.TelemetryRequest{
 		SessionToken: ctx.Request().Header.Get("X-Auth-Token"),
-		URL:          ctx.Request().RequestURI,
```

Comment:
This was added by Irappa for storing metric data in DB, please check if it got removed during pull.

Reply:
addressed